src/index.html

```diff
@@ -93,5 +125,13 @@
                 },
             };
         }
+
+        function openForm() {
```

Comment:
+1 for VanillaJS!

If you want, you can try to solve this using AlpineJS as well, but in practice it will have no benefit over your solution, apart from the fact that the code would be more aligned to the rest.